Makefile

```
@@ -229,15 +229,15 @@ gcc: apt-get-gcc
 git: apt-get-git
 jq: apt-get-jq
 psmisc: apt-get-psmisc
-python3: apt-get-python3.9
+python3: apt-get-python3.11
 tox: apt-get-tox
 unzip: apt-get-unzip
 wget: apt-get-wget
 
 java:
 	@ # java has a different apt-get package name.
 	if [[ "$$(which java)" == "" ]]; then \
-		sudo apt-get install -qqy --no-install-suggests openjdk-11-jdk; \
+		sudo apt-get install -qqy --no-install-suggests temurin-11-jdk; \
```

This should match the package installed in the Dockerfile.
```suggestion
		sudo apt-get install -qqy --no-install-suggests java-11-amazon-corretto-jdk; \
```

---

Thanks for spotting this! Fixed.